### PR TITLE
Fix clear options & add current location option

### PR DESCRIPTION
**Pull request by trevorgerhardt**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -126,14 +147,19 @@ class Geocoder extends PureComponent {
     this.select = select
   }
 
+  renderCount = 0
```
Comment (landonreed) [inline index.js]: @trevorgerhardt what's `renderCount`?

File: index.js [hunk 2]
```diff
@@ -68,56 +79,66 @@ class Geocoder extends PureComponent {
...
         }
-        this.setState({value: result})
-        this.props.onChange && this.props.onChange(value && this.options[value.value])
+      })
+      window.navigator.geolocation.getCurrentPosition((position) => {
+        reverseSearch({
+          apiKey,
+          point: position.coords
+        }).then((geojson) => {
+          const value = this.featureToOption(geojson.features[0])
```
Comment (landonreed) [inline index.js:122]: @trevorgerhardt what if there's an error in the `reverseSearch`?  Wouldn't we want to just return the `position.coords` as a fallback?

File: index.js (hunk 1 above)
Comment (trevorgerhardt) [reply index.js]: @landonreed whoops, debugging artifact

File: index.js (hunk 2 above)
Comment (trevorgerhardt) [reply index.js:122]: Good call 👌

Comment (trevorgerhardt) [reply index.js:122]: Done!

**Review by landonreed — APPROVED** [review]
LGTM!

**Pull request merged**